inlang/source-code/editor/package.json

```diff
@@ -4,7 +4,7 @@
 	"private": true,
 	"scripts": {
 		"dev": "vite",
-		"build": "paraglide-js compile --project ../../../project.inlang && vite build -- --max_old_space_size=1200000",
+		"build": "paraglide-js compile --project ../../../project.inlang && tsc && vite build -- --max_old_space_size=1200000",
 		"production": "NODE_ENV=production tsx ./src/server/main.ts",
 		"---- TEST ----------------------------------------------------------": "",
 		"test": "paraglide-js compile --project ../../../project.inlang && tsc --noEmit",
```

Comment:
we had tsc as part of build scripts if I recall correctly but Ivan Hofer moved them to test scripts to speed up builds. I have no preference for either way, just adding context

Reply:
the issue with test is that it runs after build and takes a long time to run and discover type issues

Reply:
my favorite would be to either have it part of lint, but i would be ok to have it run before build in the build step

Reply:
Ahh - now i see. tsc is part of test in most of the projects. We just didn't see it failing in the ci in manage and paraglide because those packages just doesn't have a test or don't have tsc in test. I would preferre to have it in build for the reason jan stated - tests are just to slow for knowing it it builds at least. I will create a separate pr for that since it changes are to hard to review now